pytest_reportportal/pytest_rp_plugin.py

```diff
@@ -0,0 +1,104 @@
+import pytest
```

Comment:
is it possible to add some license header? GPL-3.0 should be specified somewhere in the file